BPStatusBarAlert/Classes/DeviceDimensions.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  DeviceDimensions.swift
+//  
+//
+//  Created by Sameh Salama on 11/28/17.
+//
+
+import UIKit
+
+public class DeviceDimensions {
```

Comment:
it seemed to be more swift when change `class` to `struct` 

And, It would be better to assign this class's accessor to the internal level.

Reply:
-changed to ```struct```
-It would be better to assign this class's accessor to the internal level, *would you please explain more*